gym-service/src/main/java/org/tolking/external_dto/TrainingEventDTO.java

```diff
@@ -4,24 +4,37 @@
 import lombok.Data;
 import org.tolking.enums.ActionType;
 
+import java.io.Serial;
+import java.io.Serializable;
 import java.util.Date;
 
 @Data
-public class TrainingEventDTO {
+public class TrainingEventDTO implements Serializable {
```

Comment:
Just curious, you use Jackson mapper, does it still require to class implement serializable?

Reply:
Yeap. When tried without implementing it, it threw an error (ActiveMQ)